annif/project.py

```diff
@@ -153,9 +153,32 @@ def vocab(self):
     def subjects(self):
         return self.vocab.subjects
 
+    @property
+    def is_trained(self):
+        try:
+            return self.backend.is_trained
+        except AnnifException as err:
```

Comment:
Why catch this exception here instead of letting it bubble up to the caller?

Reply:
The reason for catching is that if project definition is missing backend, `list-projects` or `show-project` CLI commands crash. There is such a project (`nobackend`) in `tests/projects.cfg`. Maybe it would be better to catch in CLI methods (or remove that project from test projects)... 

I'll check if there are merge conflicts from #417 and look at this then again.